README.md

```diff
@@ -0,0 +1,31 @@
+Titolo
+Breve descrizione progetto
+Tecnologie utilizzate
+Come installare il progetto
+Come far partire il progetto
+Come utilizzare il progetto
+
+# Trip Planner
+
+Hello, this project consists in having an organized schedule of a trip you are planning to undertake. It allows you to filter the contents by type and was created to showcase some of the main functionalities of React.

```

Comment:
Direi qualcosa tipo: `This tool helps you view and filter the details of your next trip on both desktop and mobile.  It was created to showcase some of the main functionalities of React.`